Mathlib/Topology/Algebra/InfiniteSum/Order.lean

```diff
@@ -143,83 +146,92 @@ protected theorem Multipliable.le_tprod (hf : Multipliable f) (i : ι) (hb : ∀
 @[to_additive existing, deprecated (since := "2025-04-12")] alias le_tprod := Multipliable.le_tprod
 
 @[to_additive (attr := gcongr)]
-protected theorem Multipliable.tprod_le_tprod (h : ∀ i, f i ≤ g i) (hf : Multipliable f)
-    (hg : Multipliable g) : ∏' i, f i ≤ ∏' i, g i :=
-  hasProd_le h hf.hasProd hg.hasProd
+protected theorem MultipliableFilter.tprodFilter_le_tprodFilter (h : ∀ i, f i ≤ g i) [L.NeBot]
```

Comment:
how do I make the gcongr attribute apply to the `tsum_le_tsum` case? Do we just need a version of this lemma in that case?